code/game/gamemodes/traitor/traitor.dm

```diff
@@ -137,6 +125,14 @@
 				text += "<br><font color='red'><B>The [special_role_text] has failed!</B></font>"
 				feedback_add_details("traitor_success","FAIL")
 
+		if(length(SSticker.mode.implanted))
+			text += "<br><br><FONT size = 2><B>The mindslaves were:</B></FONT><br>"
```

Comment:
Don't they already show as traitors with no objective but 'protect'?

Reply:
They do not. Previously there was nothing shown.